class.jetpack-gutenberg.php

```diff
@@ -89,6 +94,47 @@ class Jetpack_Gutenberg {
 	 */
 	private static $availability = array();
 
+	/**
+	 * Check to see if a block is able to run with current wp or gutenburg plugin versions
+	 *
+	 * @param array  $version_requirements An array containing wp and plugin version requirements.
+	 * @param string $extension_name The name of the block/plugin that has the version requirement.
+	 *
+	 * @since 8.4.0
+	 *
+	 * @return boolean True if the version of gutenberg required by the plugin is available.
+	 */
+	public static function is_editor_version_available( $version_requirements, $extension_name ) {
+		global $wp_version;
+
+		if ( defined( 'GUTENBERG_DEVELOPMENT_MODE' ) && GUTENBERG_DEVELOPMENT_MODE ) {
+			return true;
+		}
+
+		if ( defined( 'GUTENBERG_VERSION' ) ) {
+			$version_available = GUTENBERG_VERSION >= $version_requirements['plugin'];
+		} else {
```

Comment:
What if the WordPress version check was separated from the Gutenberg plugin check, so you could check for the 2 if necessary?

Reply:
I have expanded the comments in this method to better explain why we have both the gutenberg and wp versions here. As far as I can tell, there is no way in a core wp install, without the gutenberg plugin, to get the current gutenberg version in php, so in this case we need to know the minumum core WordPress release that included the minimum Gutenberg version that we want. 

Because of the need for this fallback to the wp version it seemed to overly complicate things by pulling out the WP version check and working out if you need and AND, or an OR on the versions. It may be simpler if the need ever arises to have to check both to have a separate is_wp_version_available call. 

Open to feedback on this though, as I may be oversimplifying/overcomplicating things. It would be nice if there was a way to get the current gutenberg version from php regardless of whether plugin was installed or not. 

I have renamed the method to is_gutenberg_version_available which @scruffian suggested, to further clarify exactly what it is we are checking here.